Collaboration (#142)
XMPro/Blueprints-Accelerators-Patterns: docs/collections/_blueprints/general-aa-meta-tags.md

---
layout: project-top
date: 2024-05-03
title: "Asset Analysis Meta Tag Application"
description: "<strong>Asset Analysis Meta Tag Application</strong>"

> extract: "A Asset labelling system that allows the user to assign additional attributes to existing Asset that aid in data analysis methods."

Reviewer: *An Asset

*to an existing Asset

* that aids

Author: done

weight: 2

> thumbnail: "/assets/images/blueprints/general-ra-meta-tags/ad03.png"

Reviewer: `general-ra` or `genera-aa` ?

Author: done

> image: "/assets/images/blueprints/general-ra-meta-tags/ad03.png"

Reviewer: /
```suggestion
image: "/assets/images/blueprints/general-aa-meta-tags/ad03.png"
```

Author: done

categories: ["Data Capture","Light Theme"]
contributor: "XMPro"
role: "Blueprint"
import_password: Dem0nstr@t1on
toc: true
toc_label: "Table of Contents"

gallery:
  - image: "/assets/images/blueprints/general-aa-meta-tags/ad03.png"
    caption: "<strong>Above:</strong> Application View"
  - image: "/assets/images/blueprints/general-aa-meta-tags/ad04.png"
    caption: "<strong>Above:</strong> Reference Table View"
gallery_limit: 2

files:
  - title: "Application"
    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-aa-meta-tags/application"
    image: "/assets/images/icons/AD.png"
  - title: "SQL"
    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-aa-meta-tags/sql/"
    image: "/assets/images/icons/sql.webp"
  - title: "Datastream"

>     link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/sql/"

Reviewer: ```suggestion
    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-aa-meta-tags/sql/"
```

Author: done

    image: "/assets/images/icons/DS.png"

---

## Application

This blueprint app establishes meta tag labels for assets. This allows assets to be filtered and analysed by labels such as asset class or attached technology. 

### Create Meta Tags

<div class="inline_image">{% include framework/shortcodes/image.html src="/assets/images/blueprints/general-aa-meta-tags/ad01.png" %}</div>

> This page contains a navigation bar at the top right, a asset selection grid on the left, and create/ update card on the right. 

Reviewer: ```suggestion
This page contains a navigation bar at the top right, an asset selection grid on the left, and create/update card on the right. 
```

Author: done

> Each grid has the Store User option enabled meaning it will remember your selection for column ordering, grouping and for the bottom grid hiding certain columns.  Selecting any of the rows in either grid will drilldown to the action page for the RCA that was selected.

Reviewer: Is it RCA?

Author: oof. done

### Create Meta Tag Labels
<div class="inline_image"  >{% include framework/shortcodes/image.html src="/assets/images/blueprints/general-aa-meta-tags/ad02.png" %}</div>

Similar to the Create Meta Tags page, the user will either select a Meta Tag from the drop down and enter a unique Meta Tag Value Label or select a Meta Tag Value from the grid to edit.

A Radio Button above the grid allows users to select among active, inactive, or all Meta Tags to filter the Meta Tag Values.

### Assign Meta Tag and Label
<div class="inline_image">{% include framework/shortcodes/image.html src="/assets/images/blueprints/general-aa-meta-tags/ad03.png" %}</div>

The Assets Grid Selection is broken into Assigned, Unassigned, and All. The user will click on a specific asset to assign Meta Tag Values to the asset. Dropdown menus will then appear. If the value has already been assigned, the value will appear; if not, then the value will be empty.

> The user must assign all values to all tags before they will be allowed to save. By default, when a Meta Tag is created, a default value of Not Allocated is also created. The user should select this option if a Meta Tag is not rquired for a asset.

Reviewer: ```suggestion
The user must assign all values to all tags before they will be allowed to save. By default, when a Meta Tag is created, a default value of "Not Allocated" is also created. The user should select this option if a Meta Tag is not required for an asset.
```

Author: done

> The user must assign all values to all tags before they will be allowed to save. By default, when a Meta Tag is created, a default value of Not Allocated is also created. The user should select this option if a Meta Tag is not rquired for a asset.

Reviewer: Maybe a short explanation of why "Not Allocated" instead of "none/null/blank"

Author: "The user should select this option if a Meta Tag is not required for an asset as it signifies that user reviewed the entry and assigned a value."?
